doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -712,10 +714,13 @@ Libraries for networking
 
     * Support for using pre-provisioned X.509 certificates.
     * Support for using DTLS Connection Identifier
+    * Support for MCUmgr SMP client for external FOTA support with shell.
 
   * Updated:
 
     * Zephyr's LwM2M Connectivity Monitor object now uses a 16-bit value for radio signal strength so it does not roll over on values smaller than -126 dBm.
+    * LwM2M FOTA advanced FOTA support external MCU with DFU SMP target.
+    * Integrate new Fota download utils API for LwM2M firmware utils library.
```

Comment:
```suggestion
    * Advanced LwM2M FOTA support for an external MCU with DFU SMP target.
    * FOTA download Utils API integrated to the library.
```
Isn't the latter one rather something that is added (not updated), if the API is new?

Reply:
Ah true I will move these 2 lines to added